arches_lingo/views/trees.py

```diff
@@ -0,0 +1,363 @@
+        # TODO: sort on sortorder at higher stacklevel once captured in original data.
+        broader_concepts = sorted(self.broader_concepts[conceptid])
+        try:
+            first_broader_conceptid = broader_concepts[0]
+        except IndexError:
+            # TODO: sort here too.
+            schemes = sorted(self.schemes_by_top_concept[conceptid])
+            working_parent_list.insert(0, schemes[0])
+            return working_parent_list
+        else:
+            working_parent_list.insert(0, first_broader_conceptid)
+            return self.add_broader_concept_recursive(
+                working_parent_list, first_broader_conceptid
+            )
+
+    def serialize_concept_label(self, label_tile: dict):
+        lang_code = self.language_concepts[label_tile[CONCEPT_NAME_LANGUAGE_NODE][0]]
+        localized_string_objs = label_tile[CONCEPT_NAME_CONTENT_NODE].values()
+        try:
+            value = next(iter(localized_string_objs))["value"]
+        except (StopIteration, KeyError):
+            value = "Unknown"
+        return {
+            "valuetype": self.human_label_type(label_tile[CONCEPT_NAME_TYPE_NODE]),
+            "language": lang_code,
+            "value": value,
+        }
+
+    def get(self, request):
+        data = {
+            "schemes": [self.serialize_scheme(scheme) for scheme in self.schemes],
+        }
+        # Todo: filter by nodegroup permissions
+        return JSONResponse(data)
+
+
+@method_decorator(
+    group_required("RDM Administrator", raise_exception=True), name="dispatch"
+)
+class ValueSearchView(ConceptTreeView):
+    def get(self, request):
+        term = request.GET.get("term")
+        max_edit_distance = request.GET.get(
+            "maxEditDistance", self.default_sensitivity()
+        )
+        page_number = request.GET.get("page", 1)
+        items_per_page = request.GET.get("items", 25)
+
+        concept_query = VwLabelValue.objects.all()
+        if term:
+            concept_query = (
+                concept_query.annotate(
+                    edit_distance=LevenshteinLessEqual(
+                        F("value"),
+                        Value(term),
+                        Value(max_edit_distance),
+                        output_field=FloatField(),
+                    )
+                )
+                .filter(edit_distance__lte=max_edit_distance)
+                .order_by("edit_distance")
+            )
+        else:
+            concept_query = concept_query.order_by("concept_id")
+        concept_query = concept_query.values_list("concept_id", flat=True).distinct()
+
+        paginator = Paginator(concept_query, items_per_page)
+        page = paginator.get_page(page_number)
+
+        data = [
+            self.serialize_concept(str(concept_uuid), parents=True, children=False)
+            for concept_uuid in page
+        ]
+
+        # Todo: filter by nodegroup permissions
+        return JSONResponse(data)
```

Comment:
This has been altered in https://github.com/archesproject/arches-lingo/pull/79 to return more data about the query